cmd/run.go

```diff
@@ -75,7 +109,14 @@ func notify(cmds [][]string) {
 		runningLock.Unlock()
 	}()
 
-	for _, cmd := range cmds {
+	for _, args := range cmds {
+		env, cmd := splitEnvCmd(args)
+		backupEnv, err := setEnvVar(env)
+		if err != nil {
+			log.Error("Fail to set environment variable %v - %v", env, err)
+			fmt.Print("\x07")
+			return
+		}
 		command := exec.Command(cmd[0], cmd[1:]...)
 		command.Stdout = os.Stdout
 		command.Stderr = os.Stderr
```

Comment:
See [exec.Command.Env](https://gowalker.org/os/exec#Cmd) which allows you set env vars for a specific command without mess up wider range. May save some work of setting/restoring env vars.